app-i18n/fcitx-chinese-addons/fcitx-chinese-addons-5.1.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="fcitx5-chinese-addons"
+
+inherit cmake xdg
+
+DESCRIPTION="Addons related to Chinese, including IME previous bundled inside fcitx4."
+HOMEPAGE="https://github.com/fcitx/fcitx5-chinese-addons"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}_dict.tar.xz"
+
+LICENSE="GPL-2+ LGPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="5"
+IUSE="webengine +cloudpinyin +qt5 lua +opencc test"
```

Comment:
Maybe we should use `gui` as flag for `-DENABLE_GUI`.
QT6 support is coming in next release for fcitx5-chinese-addons.

Reply:
I reread the code for the master branch. I think I misunderstood it. I'll re-update the code.

Reply:
As qt5 is the only option in this release, I mean something like 
```
IUSE="webengine +cloudpinyin +gui lua +opencc test"

gui? (
	dev-qt/qtconcurrent:5
	app-i18n/fcitx-qt:5[qt5,-onlyplugin]
	webengine? ( dev-qt/qtwebengine:5[widgets] )
)
```
Then in next release we can have
```
IUSE="webengine +cloudpinyin +gui +qt6 lua +opencc test"

gui? (
	!qt6? (
		dev-qt/qtconcurrent:5
		app-i18n/fcitx-qt:5[qt5,-onlyplugin]
		webengine? ( dev-qt/qtwebengine:5[widgets] )
	)
	qt6? (
		qt*:6
		webengine? ( dev-qt/qtwebengine:6[widgets] )
	)
)
```
And
```
mycmakeargs=(
	...
	-DENABLE_GUI=$(usex gui)
	-DUSE_QT6=$(usex qt6)
	...
)
```

Reply:
I've mostly modified it as you suggested, but kept the qt5 use.

When the next version of fcitx5-english-addons is released, qt6 use will be added. at that time, the choice of qt5 or qt6 as default use needs to be decided based on the qt6 progress of the main tree and the default configuration of the upstream repository.